runtime/autoload/health/nvim.vim

```diff
@@ -414,15 +413,20 @@ function! s:check_ruby() abort
     let prog_vers = 'not found'
     call health#report_error('Missing Neovim RubyGem', suggestions)
   else
+    silent let latest_gem   = systemlist("gem list -ra '^neovim$' | " .
+          \ "awk -F'[()]' '{print $2}' | " .
+          \ 'cut -d, -f1')[0]
```

Comment:
If no results are found (e.g. no internet connection), `[0]` will cause 'index out of range' error. This avoids it:

```vim
let latest_gem = get(systemlist(...), 0, 'unknown')
```